app/client/nginx.conf

```diff
@@ -1,5 +1,5 @@
 server {
-  listen 8080;
+  listen 80;
```

Comment:
this needs to be 8080 please.

Reply:
or better yet, roll back the change on this file.

Reply:
Hmm okay that breaks the development side though

Reply:
@akroon3r would it work if we left nginx to listen on port 8080, Dockerfile still exposes to 80, and mapped 80:8080 in docker-compose?

Reply:
I don't see why we can't experiment with that.